server/src/logic/services/auth.service.ts

```diff
@@ -137,9 +137,11 @@ export class AuthService {
         .addPassword(hashedPassword)
         .addImage((process.env.DEFAULT_USER_IMAGE || Config.DEFAULT_USER_IMAGE))
         .addHasTermsAccepted(true)
-        .addIsAdmin(true)
+        .addIsAdmin(isAdmin)
```

Comment:
I would add an enum "UserRole" and change addIsAdmin to addUserRole or simply role!?